src/Logging/Logging.Console/src/ConsoleLoggerOptions.cs

```diff
@@ -7,6 +7,7 @@ public class ConsoleLoggerOptions
     {
         public bool IncludeScopes { get; set; }
         public bool DisableColors { get; set; }
+        public ConsoleLoggerFormat Format { get; set; }
```

Comment:
Similar doc comment suggestion here. We're trying to get new types/members off on the right foot with `///` doc comments (even though we have a bunch of gaps in existing ones 😦)